src/index.html

```diff
@@ -3,10 +3,457 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://static-00.iconduck.com/assets.00/cosmos-cryptocurrency-icon-2048x2048-k2k2lftp.png"
+    rel="icon" type="image/x-icon">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
```

Comment:
The stylesheet link should have a `.css` file extension instead of `.scss`. Change the href value to './styles/main.css'